js/src/modals/ExecuteContract/DetailsStep/detailsStep.js

```diff
@@ -141,6 +142,24 @@ export default class DetailsStep extends Component {
           );
           break;
 
+        case 'bool':
+          const boolitems = ['false', 'true'].map((bool) => {
+            return (
+              <MenuItem
+                key={ bool }
+                value={ bool }
+                label={ bool }>{ bool }</MenuItem>
+            );
+          });
+          inputbox = (
+            <Select
```

Comment:
I was kinda expecting such reason. I think that for now, the dropdown is functional and integrates well in the UI.